site/src/components/Resources/Resources.tsx

```diff
@@ -102,9 +106,11 @@ export const Resources: FC<ResourcesProps> = ({
                         {agent.name}
                         <div className={styles.agentInfo}>
                           <span className={styles.operatingSystem}>{agent.operating_system}</span>
-                          <span style={{ color: agentStatus.color }} className={styles.status}>
-                            {agentStatus.status}
-                          </span>
+                          {WorkspaceStateEnum[workspaceStatus] !== "Stopped" && (
```

Comment:
what if we used the Enum to get the value?
```suggestion
                          {WorkspaceStateEnum[workspaceStatus] !== WorkspaceStateEnum["stopped"] && (
```

Reply:
oh yeah hahaha thanks!